src/main/java/mate/academy/patternsOnLesson/SingletonPattern.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.patternsOnLesson;
+
+public enum SingletonPattern implements Pattern {
+    INSTANCE;
```

Comment:
Where is INSTANCE initializing?

Reply:
![image](https://user-images.githubusercontent.com/41755893/50522676-d4bcd380-0ad4-11e9-8bcc-63c017ab5f82.png)
here) or I don't understand.

Reply:
INSTANCE on the picture is just an example. You need to initialize it. What is the data type of your INSTANCE? Is it String, Object or maybe some your custom class like User, Car ....

Reply:
![image](https://user-images.githubusercontent.com/41755893/50525717-efe40f00-0ae5-11e9-9dfa-c86184ae3921.png)
![image](https://user-images.githubusercontent.com/41755893/50525726-fbcfd100-0ae5-11e9-8584-cc21ad0e90e8.png)